packages/react/src/components/DataTable/__tests__/DataTable-test.js

```diff
@@ -208,6 +208,31 @@ describe('DataTable', () => {
           'Field 3:B',
         ]);
       });
+      it('should works correctly with `initialSortColumn` and ` initialSortDirection` props', () => {
```

Comment:
Not able to figure out why test is failing 🤔 
props are working correctly in storybook

Reply:
@tw15egan @tay1orjones  can u please look into this when u have some time

Reply:
Is the table sorted as expected on the first render? If not, it might be a re-render causing the tests to fail.